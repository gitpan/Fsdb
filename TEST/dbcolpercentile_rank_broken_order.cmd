prog='dbcolpercentile'
args='-S --rank test1'
in=TEST/dbcolhisto_ex.in
cmp='diff -cb '
expected_exit_code=fail
