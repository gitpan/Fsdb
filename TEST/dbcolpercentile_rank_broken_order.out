dbcolpercentile: data out of order between 65 and 90, should be in descending order.
 at blib/script/dbcolpercentile line 187
#fsdb name id test1 rank
a	1	80	1
b	2	70	2
c	3	65	3
